packages/babel-plugin-transform-es2015-classes/test/fixtures/loose/derived-constructor-no-super-return-falsey/expected.js

```diff
@@ -0,0 +1,11 @@
+var Child = function (_Base) {
+  babelHelpers.inheritsLoose(Child, _Base);
+
+  function Child() {
+    var _this;
```

Comment:
maybe we don't have to do this now, but the _this seems unnecessary now?

Reply:
It'd be a lot more work to get rid of the `thisRef` stuff. Really, this should be a throw expression when used.